crates/turborepo-lib/src/shim.rs

```diff
@@ -474,10 +477,20 @@ impl RepoState {
                 }
 
                 // FIXME: We should save this package manager that we detected
-                let workspace_globs =
-                    PackageManager::get_package_manager(path, package_json.as_ref())
-                        .and_then(|mgr| mgr.get_workspace_globs(path))
-                        .ok();
+                let workspace_globs = match PackageManager::get_package_manager(path, None)
```

Comment:
Do we not want to include the `package_json` here? We're now relying running detection even if they specified a package manager via `packageManager`

Reply:
I think that was a leftover from trying to trace a fix. Definitely not supposed to be in the commit.